experimental/sdk/webpubsub-socketio-extension/src/common/utils.ts

```diff
@@ -92,6 +93,30 @@ export function getWebPubSubServiceCaller(
   }
 }
 
+export function getWebPubSubServiceClient(
+  options: WebPubSubExtensionOptions | WebPubSubExtensionCredentialOptions
+): WebPubSubServiceClient {
+  debug(`getWebPubSubServiceClient, ${JSON.stringify(options)}`);
```

Comment:
maybe extract some method as `IsCredentialOption` later

Reply:
I'll do this later